docs/internal/Element-Type-Specs.md

```diff
@@ -253,6 +253,7 @@ _Multi-choice question, with one allowed option, displayed as Drop-down list (Co
   optionIndex: <integer> (index from the options array)
   text: <string> (actual text from options array)
   selection: <string | object> (entire object or string from the supplied options list)
+  isCustomOption: <boolean> (if the selection is a custom "hasOther" option)
```

Comment:
Not essential, but thought it might be useful to indicate in the response whether the selection is a custom value or not. An action might do something different depending on this.